subsys/mgmt/ec_host_cmd/Kconfig

```diff
@@ -81,6 +81,12 @@ config EC_HOST_CMD_INITIALIZE_AT_BOOT
 	  Automatically initialize the host command subsystem with the chosen backend.
 	  The ec_host_cmd_init function is called by the chosen backend.
 
+config EC_HOST_CMD_DEDICATED_THREAD
+	bool "Create a new task for host command"
+	help
+	  Create a new thread dedicated for Host Command during initialization.
```

Comment:
I find the "during initialization" bit somewhat ambiguous since the thread is started by a function called by SYS_INIT on a different file. Should this be rephrased somehow?

Reply:
Done.